src/hb-ot-layout.cc

```diff
@@ -60,6 +61,9 @@ _hb_ot_layout_create (hb_face_t *face)
   layout->gpos_blob = OT::Sanitizer<OT::GPOS>::sanitize (face->reference_table (HB_OT_TAG_GPOS));
   layout->gpos = OT::Sanitizer<OT::GPOS>::lock_instance (layout->gpos_blob);
 
+  layout->cpal_blob = OT::Sanitizer<OT::CPAL>::sanitize (face->reference_table (HB_OT_TAG_CPAL));
+  layout->cpal = OT::Sanitizer<OT::CPAL>::lock_instance (layout->cpal_blob);
+
```

Comment:
I’m just wondering if we should load the `CPAL` table lazily like #241 does for the `MATH` table, though I’m not sure if it makes any practical difference.